Helper/Data.php

```diff
@@ -63,6 +63,8 @@ public function getConfig($field, $default="", $store = null)
         if($result == "") {
             $result = $default;
         }
+		

```

Comment:
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.